packages/apps/tools/src/components/Global/FormStatusNotification/index.tsx

```diff
@@ -79,11 +79,12 @@ export const FormStatusNotification: FC<IFormStatusNotificationProps> = (
         color={statusToColorMap[status!]}
         hasCloseButton
         icon={statusToIconMap[status!]}
-        expanded
-        title={title ?? titles[status!]}
         onClose={onNotificationCloseClick}
-        variant="outlined"
+        role="status"
       >
+        {title ? (
```

Comment:
This is not the correct replacement for `title ?? titles[status!]`

Reply:
I think my brain registered && haha